small-talk/app/chat/page.test.js

```diff
@@ -0,0 +1,65 @@
+const React = require('react');
+const { render, fireEvent, waitFor } = require('@testing-library/react');
+const Chat = require('../chat/page.js');
+const socket = require('../../util/socket');
+
+// Mock socket.io module
+jest.mock('../../util/socket', () => ({
+    auth: null,
+    connect: jest.fn(),
+    on: jest.fn(),
+    off: jest.fn(),
+    emit: jest.fn(),
+    id: 'mockedSocketID'
+}));
+
+// create all the test cases for what is currently up in the page.js, can add more for future functionality 
+describe('Chat Component', () => {
+    // once each test case is done, clear it so the next test case is run off a fresh implementation
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    // check that the chat page loads without crashing or taking too long
+    it('renders without crashing', () => {
+        render(<Chat />);
+    });
```

Comment:
Are there no other checks needed here after the render? I would assume it would just fail if there was an error simply rendering the Chat but I'm just not sure.

Reply:
I was also thinking the same and if we do need more checks we can simply add it after changes have been made to the chatroom.